src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.cs

```diff
@@ -445,7 +445,7 @@ public void TestCapacityReservation()
             TestRunner.RunTestScript("Test-CapacityReservation");
         }
 
-        [Fact(Skip = "Cannot run on Github Platform due to ACL API")]
+        [Fact]
```

Comment:
add in the PR description what the permission value is being changed from and what it will be now going forward for better record keeping. 
Otherwise, after offline discussion, looks ok.

Reply:
Added